### ISSUE TITLE
Syncplay window dragging lags

### ISSUE THREAD

**Issue by martinorav** [Syncplay/syncplay#410]
When dragging the syncplay window around, it lags. The issue is not performance, since no other window does this. I am on Windows 10, Syncplay version is 1.6.7 release 94. I'm using VLC. Video below.

https://user-images.githubusercontent.com/53046664/114277050-28e3eb80-9a32-11eb-94d7-cf520b1b1d58.mp4

**Comment by Et0h** [comment]
It's not something I can reproduce on my machine so I'm probably not the right person to try to fix it.

Some questions:
1. Are you running Syncplay in an anti-virus sandbox which is causing the slowdown?
2. Are you using a multi-monitor setup? If so, are both monitors operating at the same refresh rate?
3. Is this a high-spec machine?

**Comment by martinorav** [comment]
> It's not something I can reproduce on my Windows 10 machine so I'm probably not the right person to try to fix it.
> 
> Some questions:
> 
> 1. Are you running Syncplay in an anti-virus sandbox which is causing the slowdown?
> 2. Are you using a multi-monitor setup? If so, are both monitors operating at the same refresh rate?
> 3. Is this a high-spec machine?
> 4. Does the issue occur if you use mpv rather than VLC as your media player?

1. Not that I know of, so unless its something windows does automatically, no.
2. Yes. Yes. Also, the refresh rate in question is 144 hz, however the issue also happened to a friend who I think has only 60 hz monitors (I'll update for certain when I get a reply). It also did not happen to another friend, who has multiple monitors with 60 hz refresh rates.
3. i7-9750H, RTX 2070 Max-Q laptop, however the issue also happened to a friend, who has a desktop computer, so its unlikely to be an issue with power limitations.
4. No, but during testing this I discovered, that:
   1) When using MPV, it works perfectly most of the time, but sometimes just lags, like I was running at ~5 fps (notice, that just running at 5 fps is a bit different from the original issue). I can both cause and fix this by fidgeting with alt tab and dragging both the MPV and Syncplay windows around. I was not able to reproduce it consistently.
   2) The issue does not occur right after starting Syncplay. It instead starts when I actually add the video to be played into VLC. 
The issue persists, when stopping the playback by clicking the stop playback button.

**Comment by daniel-123** [comment]
I've tested this on handful of Linux systems and haven't seen the issue in any of the cases I tried either with mpv or with vlc: Intel HD 630 + 60Hz 1080p monitor and Radeon R9 380X + 75Hz 1440p monitor.

That said - from my experience I'd suspect that this *might* have something to do with power states of GPUs and different rendering paths used by different software. For example it's pretty common issue where several displays or single display at high refresh rate will result in GPU running at high power state (and if you force low power state you'll experience display artifacts).
